perception/image_projection_based_fusion/include/image_projection_based_fusion/utils/geometry.hpp

```diff
@@ -60,6 +60,10 @@ void transformPoints(
   const std::vector<Eigen::Vector3d> & input_points, const Eigen::Affine3d & affine_transform,
   std::vector<Eigen::Vector3d> & output_points);
 
+bool is_inside(
+  const sensor_msgs::msg::RegionOfInterest & outer,
+  const sensor_msgs::msg::RegionOfInterest & inner, const double outer_offset_scale = 1.1);
```

Comment:
I would like to change scale factor by launch parameter.

Reply:
updated in https://github.com/autowarefoundation/autoware.universe/pull/4147/commits/9fa0df5259dc534859c4b3beda72ab09640a1acd

Reply:
LGTM]